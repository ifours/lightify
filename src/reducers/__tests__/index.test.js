import reducer from '../index';

describe('Root reducer', () => {
  it('should return backbone of the application state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot();
  });
});
