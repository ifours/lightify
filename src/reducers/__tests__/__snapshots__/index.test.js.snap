// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root reducer should return backbone of the application state 1`] = `
Object {
  "entities": Object {
    "albums": Object {},
    "playlists": Object {},
    "tracks": Object {},
  },
  "featured": Object {
    "message": "",
    "playlists": Object {
      "items": Array [],
    },
  },
  "session": Object {
    "accessToken": undefined,
    "date": 0,
    "expiresIn": 0,
  },
}
`;
